frontend/src/components/SpecialistsList/SpecialistsList.tsx

```diff
@@ -19,6 +21,21 @@ export const SpecialistsList: FunctionComponent<Props> = props => {
             });
     }, [props.serviceId]);
 
+    const requestServiceHandler = (requestedServiceId: number) => {
+        if (user === null) {
+            throw new Error('User is null');
+        }
+
+        makeServiceRequest(user, { requestedServiceId })
+            .then(() => {
+                toast.success('Service request made successfully');
+            })
+            .catch(error => {
+                console.log(error);
```

Comment:
Should we keep the console.log(error); here ? We can at least make it console.error(error); I think.